### ISSUE TITLE
UnicodeEncodeError: 'ascii' codec can't encode characters in position 8-13: ordinal not in range(128)

### ISSUE THREAD

**Issue by ahbon123** [spyder-ide/spyder#10311]
## Description

**What steps will reproduce the problem?**

run `spyder3` in command

**Please provide any additional information below**

  File "/usr/lib/python3/dist-packages/spyder/widgets/explorer.py", line 144, in icon
    if osp.isdir(fname):
  File "/usr/lib/python3.6/genericpath.py", line 42, in isdir
    st = os.stat(s)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 8-13: ordinal not in range(128)
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/spyder/widgets/explorer.py", line 144, in icon
    if osp.isdir(fname):
  File "/usr/lib/python3.6/genericpath.py", line 42, in isdir
    st = os.stat(s)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 8-13: ordinal not in range(128)
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/spyder/widgets/explorer.py", line 144, in icon
    if osp.isdir(fname):
  File "/usr/lib/python3.6/genericpath.py", line 42, in isdir
    st = os.stat(s)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 8-13: ordinal not in range(128)
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/spyder/widgets/explorer.py", line 144, in icon
    if osp.isdir(fname):
  File "/usr/lib/python3.6/genericpath.py", line 42, in isdir
    st = os.stat(s)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 8-13: ordinal not in range(128)
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/spyder/widgets/explorer.py", line 144, in icon
    if osp.isdir(fname):
  File "/usr/lib/python3.6/genericpath.py", line 42, in isdir
    st = os.stat(s)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 8-13: ordinal not in range(128)
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/spyder/widgets/explorer.py", line 144, in icon
    if osp.isdir(fname):
  File "/usr/lib/python3.6/genericpath.py", line 42, in isdir
    st = os.stat(s)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 8-13: ordinal not in range(128)
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/spyder/widgets/explorer.py", line 144, in icon
    if osp.isdir(fname):
  File "/usr/lib/python3.6/genericpath.py", line 42, in isdir
    st = os.stat(s)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 8-13: ordinal not in range(128)
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/spyder/widgets/explorer.py", line 144, in icon
    if osp.isdir(fname):
  File "/usr/lib/python3.6/genericpath.py", line 42, in isdir
    st = os.stat(s)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 8-13: ordinal not in range(128)

## Version and main components

* Spyder Version: 3.2.6 
* Python Version: 3.6.8
* Qt Versions: 5.9.5, PyQt5 5.10.1 on Linux

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.4.2 (OK)
pandas >=0.13.1  :  0.24.2 (OK)
numpy >=1.7      :  1.16.2 (OK)
sphinx >=0.6.6   :  2.2.0 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.15.1 (OK)
psutil >=0.3     :  5.4.2 (OK)
nbconvert >=4.0  :  5.6.0 (OK)
sympy >=0.7.3    :  None (NOK)
cython >=0.21    :  None (NOK)
qtconsole >=4.2.0:  4.5.5 (OK)
IPython >=4.0    :  5.5.0 (OK)
pylint >=0.25    :  1.8.3 (OK)

```

**Comment by ccordoba12** [comment]
This is probably fixed in our latest version (3.3.6). Please update and try again.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
